cmd/fluxd/main.go

```diff
@@ -60,7 +59,7 @@ const (
 
 	// This is used as the "burst" value for rate limiting, and
 	// therefore also as the limit to the number of concurrent fetches
-	// and memcached connections, since these in general can't do any
+	// and cache backend connections, since these in general can't do any
```

Comment:
I don't think it's used for Redis though?

Reply:
indeed, but we could use the same option for the `cache.RedisConfig` __MaxConns__ property: WDYT?